### PR TITLE
Synthetic dataset generation with PROMPTGATOR

### PR DESCRIPTION
**Pull request by stalkermustang**
Based of paper Promptagator: Few-shot Dense Retrieval From 8 Examples
https://arxiv.org/abs/2209.11755

### CODE REVIEW

**Comment by stalkermustang** [comment]
Hey @simonw , it seems I have an issue here with LaTeX rendering. Could you please help me? I don't know how to allow .mdx to render the formula. Tried to find an example in the repo, but it seems no one used latex. Is there a workaround? (other than converting everything in .md) 

cc @iGusev @napo @S4MFI @Denny-Hwang as top contributors

File: pages/applications/synthetic_rag.en.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+*Output:*
+```
+punishment house would make fines relative income
+```
+
+In general, such a prompt can be expressed as:
+
+$(e_{prompt}, e_{doc}(d_{1}), e_{query}(q_1), . . . , e_{doc}(d_k), e_{query}(q_k), e_{doc}(d))$
+
+, where $e_{doc}$ and $e_{query}$ are task-specific document, query descriptions respectively, $e_{prompt}$ is a task-specific prompt/instruction for ChatGPT/GPT-4, and $d$ is a new document, for which LLM will generate a query.
```
Comment (stalkermustang) [inline pages/applications/synthetic_rag.en.mdx:64]: Any ideas on how to make this render properly in .mdx?

File: pages/applications/synthetic_rag.en.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+*Output:*
+```
+punishment house would make fines relative income
+```
+
+In general, such a prompt can be expressed as:
+
+$(e_{prompt}, e_{doc}(d_{1}), e_{query}(q_1), . . . , e_{doc}(d_k), e_{query}(q_k), e_{doc}(d))$
```
Comment (stalkermustang) [inline pages/applications/synthetic_rag.en.mdx:62]: Any ideas on how to make this render properly in .mdx?

**Comment by stalkermustang** [comment]
Hey @omarsar, just wanted to understand if there's something wrong with this PR. Could you please share your thoughts for this and #298 ?

**Comment by omarsar** [comment]
@stalkermustang I will take a look either today or tomorrow. Thanks. I have also noticed the second part too. Will be taking a closer look at both. thanks

**Pull request merged**